@coalman321 Unfortunately the CI failed on Windows with the following errors:
```
C:\ci\ws\src\coalman321\rosbag2\rosbag2_cpp\src\rosbag2_cpp\reindexer.cpp(194,55): error C2678: binary '<': no operator found which takes a left-hand operand of type 'std::chrono::system_clock::time_point' (or there is no acceptable conversion) [C:\ci\ws\build\rosbag2_cpp\rosbag2_cpp.vcxproj]

C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include\system_error(191,28): message : could be 'bool std::operator <(const std::error_code &,const std::error_code &) noexcept' [found using argument-dependent lookup] (compiling source file C:\ci\ws\src\coalman321\rosbag2\rosbag2_cpp\src\rosbag2_cpp\reindexer.cpp) [C:\ci\ws\build\rosbag2_cpp\rosbag2_cpp.vcxproj]

C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include\system_error(277,28): message : or       'bool std::operator <(const std::error_condition &,const std::error_condition &) noexcept' [found using argument-dependent lookup] (compiling source file C:\ci\ws\src\coalman321\rosbag2\rosbag2_cpp\src\rosbag2_cpp\reindexer.cpp) [C:\ci\ws\build\rosbag2_cpp\rosbag2_cpp.vcxproj]

C:\ci\ws\src\coalman321\rosbag2\rosbag2_cpp\src\rosbag2_cpp\reindexer.cpp(194,55): message : while trying to match the argument list '(std::chrono::system_clock::time_point, std::chrono::time_point<std::chrono::steady_clock,std::chrono::nanoseconds>)' [C:\ci\ws\build\rosbag2_cpp\rosbag2_cpp.vcxproj]

C:\ci\ws\src\coalman321\rosbag2\rosbag2_cpp\src\rosbag2_cpp\reindexer.cpp(195,53): error C2679: binary '=': no operator found which takes a right-hand operand of type 'std::chrono::time_point<std::chrono::steady_clock,std::chrono::nanoseconds>' (or there is no acceptable conversion) [C:\ci\ws\build\rosbag2_cpp\rosbag2_cpp.vcxproj]
```

--- a/rosbag2_cpp/src/rosbag2_cpp/reindexer.cpp
+++ b/rosbag2_cpp/src/rosbag2_cpp/reindexer.cpp
@@ -162,8 +162,8 @@ void Reindexer::aggregate_metadata(
   const rosbag2_storage::StorageOptions & storage_options)
 {
   std::map<std::string, rosbag2_storage::TopicInformation> temp_topic_info;
-  std::chrono::system_clock::time_point latest_log_start =
-    std::chrono::system_clock::time_point::min();
+  std::chrono::time_point<std::chrono::high_resolution_clock> latest_log_start =
+    std::chrono::time_point<std::chrono::high_resolution_clock>::min();
 
   // In order to most accurately reconstruct the metadata, we need to
   // visit each of the contained relative files files in the bag,

